wire-ios-sync-engine/Source/UserSession/ZMUserSession.swift

```diff
@@ -550,6 +550,17 @@ public class ZMUserSession: NSObject {
         }
     }
 
+    private func startObservingMLSConversationVerificationStatus() {
+        guard e2eiFeature.isEnabled else {
+            return
+        }
+        Task {
+            for try await groupID in mlsService.epochChanges() {
+                try await mlsConversationVerificationStatusProvider?.updateStatus(groupID)
+            }
+        }
+    }
```

Comment:
Ok create, but now move this function into some "manager" class that makes it a lot easier to unit test since you don't need to deal with the whole user session.